example/package.json

```diff
@@ -14,14 +14,15 @@
     "babel-preset-es2015": "^6.24.0",
     "babel-preset-react": "^6.23.0",
     "cf-component-button": "^4.2.0",
-    "cf-style-const": "^0.2.0",
+    "cf-style-const": "^0.3.0",
     "cf-style-container": "^0.2.0",
     "fela": "^4.3.2",
     "fela-font-renderer": "^4.3.2",
     "fela-plugin-fallback-value": "^4.3.2",
     "fela-plugin-lvha": "^4.3.2",
     "fela-plugin-prefixer": "^4.3.2",
     "fela-plugin-unit": "^4.3.2",
+    "react": "^15.4.2",
```

Comment:
Any reason not to support the other versions?
`"react": "^0.14.0 || ^15.0.0-0 || ^16.0.0-0"`